convert2rhel/subscription.py

```diff
@@ -569,7 +582,7 @@ def attach_subscription():
 
     # check if SCA is enabled
     output, _ = utils.run_subprocess(["subscription-manager", "status"], print_output=False)
-    if "content access mode is set to simple content access." in output.lower():
+    if is_sca_enabled():
```

Comment:
The status call here can be removed right?

Reply:
Good catch I will remove it